gui/wxpython/vnet/widgets.py

```diff
@@ -511,7 +511,7 @@ def ShowColumn(self, colName, pos):
         :return: True if column was shown
         :return: False if position is not valid or column is not hidden
         """
-        if pos < 0 and pos >= self.self.GetColumnCount():
+        if pos < 0 or pos >= self.GetColumnCount():
```

Comment:
? This seems to be a slightly different change. Is it a bug fix?

Reply:
Yes, it is a fix. `self.self` here is the only place in the code base that it appears. Then, following what supplies GetColumnCount, it is https://docs.wxpython.org/wx.ListCtrl.html#wx.ListCtrl.GetColumnCount. It cannot return a negative (I really checked hard)

So, that meant that the if check would never be entered (to `continue`). So that means that the entire line must be some kind of buggy/not well written, further confirming that `self.self` wasn't intended. In the same line of thought, the condition to skip should then be if pos is under 0 or if pos is greater or equal to the column count, as that would mean that the index that will be used (the pos variable), would be out of range.